### PR TITLE
Remove grouping

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by erikzhang** [comment]
What do you think? @shargon @roman-khimov @ixje

File: src/neo/Ledger/StorageKey.cs [hunk 1]
```diff
@@ -10,31 +11,22 @@ public class StorageKey : IEquatable<StorageKey>, ISerializable
...
+            return buffer;
         }
 
+        //If the base stream of the reader doesn't support seeking, a NotSupportedException is thrown.
+        //But StorageKey never works with NetworkStream, so it doesn't matter.
         void ISerializable.Deserialize(BinaryReader reader)
         {
             Id = reader.ReadInt32();
-            Key = reader.ReadBytesWithGrouping();
+            Key = reader.ReadBytes((int)(reader.BaseStream.Length - reader.BaseStream.Position));
```
Comment (shargon) [inline src/neo/Ledger/StorageKey.cs:29]: if we have a max key of 64, and the counter is 4 bytes (integer), we can read this length into a buffer, and trim according to the `Read` return, in this way, we don't need to check the `Length` of the stream.

Comment (erikzhang) [reply src/neo/Ledger/StorageKey.cs:29]: If it is a network stream, you will consume the next message. If it isn't a network stream, we can read the length.

**Comment by shargon** [comment]
The code and the change looks good, easier to understand for developers.

**Pull request merged**